opencontainers/version.py

```diff
@@ -4,7 +4,7 @@
 # Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
 # with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-__version__ = "0.0.13"
+__version__ = "0.0.14"
```

Comment:
Perfecto! Could you add a line to the CHANGELOG to correspond with this version?

Reply:
done!